app/controllers/cart_controller.rb

```diff
@@ -22,6 +22,34 @@ def remove_item
     redirect_to '/cart'
   end
 
+  def increment
+    item = Item.find(params[:item_id])
+    cart.contents.each do |item_id, quantity|
+      if item.id.to_s == item_id
+        if quantity < item.inventory
+          cart.contents[item_id] += 1
+        else
+          flash[:inventory_notice] = "Do not have enough of this item in stock, please choose another."
+        end
+      end
+    end
+    redirect_to '/cart'
+  end
+
+  def decrement
```

Comment:
Nice logic. Like dan said maybe we need to look at being more restful at some point.